js/app/app.js

```diff
@@ -17,5 +17,60 @@ document.addEventListener("DOMContentLoaded", () => {
 })
 
 
+//** contributors section */
+
+/** ### async func get list contributors for repo from github */
+const getContributors = async (repoName) => {  
+    let request = await fetch(`https://api.github.com/repos/${repoName}/contributors`, {
+        method: 'GET',
+        headers: {
+            'Content-Type': 'application/json',
+        }
+    });
+
+    let contributorsList = await request.json();
```

Comment:
Use `const` when the value does not change.

Reply:
Done